tests/test_cram.py

```diff
@@ -0,0 +1,200 @@
+                    b'G0C1T0A1G0C1T0A1G0C1T0A1G0\tNM:i:63\n'
+            },
+            'CHROMOSOME_V': {
+                'expected_output':
+                    b'V\t16\tCHROMOSOME_V\t2\t1\t27M1D73M\t*\t0\t0\tCCTAGCCCTAACCCTAACCCTAAC'
+                    b'CCTAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAA'
+                    b'GCCTAA\t#############################@B?8B?BA@@DDBCDDCBC@CDCDCCCCCCCCC'
+                    b'CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC\tXG:i:1\tXM:i:5\tXN:i:0\tXO:i:1'
+                    b'\tXS:i:-18\tAS:i:-18\tYT:Z:UU\tMD:Z:0A0A1T0C1T0A0A0G0C1T0A0A0G0C1T0A0A'
+                    b'0G0C1T0A0A0^G0C0C0T0A0A0G0C0C0T0A0A0G0C0C0T0A0A0G0C0C0T0A0A0G0C0C0T0A0'
+                    b'A0G0C0C0T0A0A0G0C0C0T0A0A0G0C0C0T0A0A0G0C0C0T0A0A0G0C0C0T0A0A0G0C0C0T0'
+                    b'A0A0G0C0C0T0A0A0G0C0\tNM:i:96\n'
+                    b''
+                    b'VI\t0\tCHROMOSOME_V\t10\t1\t7S20M1D23M10I30M10S\t*\t0\t0\tAGCCTAAGCCTA'
+                    b'AGCCTAAGCCTAAGCTAAGCCTAAGCCTAAGCCTAAGCTTTTTTTTTTCTAAGCCTAAGCCTAAGCCTAA'
+                    b'GCCTAAGCCTAAGCCTAA\t*\tMD:Z:0A0G1C0T1A0G1C0T1A0G1C0T1A0G0^C0C0T1A0G1C0'
+                    b'T1A0G1C0T1A0G1C0T1A0G1C0T1A0G1C0T1A0G1C0T1A0G1C0T1A0G1C0T1A0G0\tNM:i:6'
+                    b'1\n'
+                    b''
+                    b'VI\t256\tCHROMOSOME_V\t10\t1\t7S20M1D23M10I30M10S\t*\t0\t0\tNNNNNNNAGC'
+                    b'CTAAGCCTAAGCCTAAGCTAAGCCTAAGCCTAAGCCTAAGNNNNNNNNNNCCTAAGCCTAAGCCTAAGCC'
+                    b'TAAGCCTAAGNNNNNNNNNN\t*\tMD:Z:20^C53\tNM:i:11\n'
+            },
+            # this one doesn't exist in the file
+            'CHROMOSOME_VI': {
+                'expected_output':
+                    b''
+            },
+        }
+
+    @classmethod
+    def tearDownClass(cls) -> None:
+        for file in cls.clean_up:
+            if os.path.exists(file):
+                os.remove(file)
+
+    def assert_cram_view_with_no_regions_generates_identical_output(self, cram_uri, crai_uri):
+        # use samtools to create a cram file from a cram file with no regions specified
+        cram_output = cram.view(cram=cram_uri, crai=crai_uri, regions=None, cram_format=True)
+        self.clean_up.append(cram_output)
+
+        # view the INPUT cram as human readable
+        cmd = f'samtools view {self.cram_local_path} -X {self.crai_local_path}'
+        p = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        expected_sam_stdout, _ = p.communicate()
+
+        # view the OUTPUT cram as human readable
+        cmd = f'samtools view {cram_output} -X {self.crai_local_path}'
+        p = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        sam_stdout, _ = p.communicate()
+
+        # check that they are the same and that the length is not zero
+        self.assertTrue(expected_sam_stdout == sam_stdout)
+        self.assertTrue(len(sam_stdout) == 2588)
```

Comment:
Since we can expect size changes in the future, consider checking that the output size is larger than 10 bytes, or something. This will make the test less brittle without significantly effecting test quality.

Reply:
This is measuring the human readable output and should not be affected by compression or changes in samtools.

Reply:
Size will change if:
  - the format  of samtools human readable output changes
  - we change fixtures

It's not clear what is accomplished with a specific size assertion.